src/internal/restapi/middleware/cors.go.tmpl

```diff
@@ -0,0 +1,40 @@
+package middleware
+
+import (
+	// "github.com/go-http-utils/headers"
+	"net/http"
+	"log"
+)
+
+func createCorsHeadersHandler(next http.Handler) func(w http.ResponseWriter, r *http.Request) {
+	handlerFunc := func(w http.ResponseWriter, r *http.Request) {
+		// ctx := r.Context()
+
+		// Example for cors middleware
```

Comment:
All our services need the cors headers for easy local development. Should be restored.

Reply:
- [ ] restore CORS logic